### ISSUE TITLE
你好，最新版本2.6.5 拉不下来 ，github 显示你的密钥过期 ，你能看看吗。

### ISSUE THREAD

**Issue by bageshishuaige** [tigerAndBull/TABAnimated#233]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
done

**Comment by bageshishuaige** [comment]
非常感谢您&nbsp; 您的库很好用&nbsp; 我一直在学习你写的三方库&nbsp;&nbsp;




くぺFOㄟAMヘ
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "tigerAndBull/TABAnimated"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月18日(星期四) 中午11:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [tigerAndBull/TABAnimated] 你好，最新版本2.6.5 拉不下来 ，github 显示你的密钥过期 ，你能看看吗。 (Issue #233)





 
done
